### PR TITLE
Track JS errors on gatsbyjs.org with Sentry

### PR DESCRIPTION
**Pull request by KyleAMathews**
/cc @Moocar

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -2718,8 +2718,6 @@
     - Styling with SCSS
     - Offline support
     - Web App Manifest
-- url: https://gatsby-starter-material-business-markdown.netlify.com/
```
Comment (KyleAMathews) [inline docs/starters.yml]: this was breaking the build

File: www/gatsby-config.js [hunk 2]
```diff
@@ -214,6 +215,7 @@ module.exports = {
               {
                 allMdx(
                   sort: { order: DESC, fields: [frontmatter___date] }
+                  limit: 10,
```
Comment (pieh) [inline www/gatsby-config.js]: Is this intentional?

File: www/gatsby-config.js [hunk 3]
```diff
@@ -282,6 +284,16 @@ module.exports = {
...
       },
     },
+    {
+      resolve: `gatsby-plugin-sentry`,
+      options: {
+        dsn: `https://2904ad31b1744c688ae19b627f51a5de@sentry.io/1471074`,
+        release: git.long(),
+        environment: process.env.NODE_ENV,
+        enabled: (() =>
+          [`production`, `stage`].indexOf(process.env.NODE_ENV) !== -1)(),
```
Comment (pieh) [inline www/gatsby-config.js]: is IIFE needed here?

File: www/gatsby-config.js (hunk 2 above)
Comment (KyleAMathews) [reply www/gatsby-config.js]: Yeah, rss feeds typically only show the ten newest posts and rendering the MDX here takes a long time as it's rendering every blog post.

File: www/gatsby-config.js (hunk 3 above)
Comment (KyleAMathews) [reply www/gatsby-config.js]: I didn't test it but the idea I believe is to only send a true or false to the browser not a function (which wouldnt work anyways).

Comment (pieh) [reply www/gatsby-config.js]: wouldn't
```js
enabled: [`production`, `stage`].indexOf(process.env.NODE_ENV) !== -1)
```
do the same? ;)

Comment (KyleAMathews) [reply www/gatsby-config.js]: Yeah, I just copied the README sample code

Comment (DSchau) [reply www/gatsby-config.js]: We should probably change the README 😅

**Pull request merged**